test/mapsproto2/combos/both/mapsproto2.pb.go

```diff
@@ -2993,9 +3003,9 @@ func (this *AllMaps) String() string {
 		keysForStringToFloatMap = append(keysForStringToFloatMap, k)
 	}
 	github_com_gogo_protobuf_sortkeys.Strings(keysForStringToFloatMap)
-	mapStringForStringToFloatMap := "map[string]*float32{"
+	mapStringForStringToFloatMap := "map[string]float32{"
 	for _, k := range keysForStringToFloatMap {
-		mapStringForStringToFloatMap += fmt.Sprintf("%v: %v,", k, this.StringToFloatMap[k])
+		mapStringForStringToFloatMap += fmt.Sprintf("%#v: %#v,", k, this.StringToFloatMap[k])
```

Comment:
%v calls String and %#v calls GoString
We want the String method to recursively call String and not GoString